Missions/TheLocalPub/LamentinLosers.conf

```
@@ -0,0 +1,13 @@
+SCR_MissionHeader {
+ World "{E223CEC24B18A917}worlds/TheLocalPub/LamentinLosers/LamentinLosers.ent"
+ m_sName "COTVT45 Lamentin Losers"
```

New naming convention per the readme is "TYPE (MIN-MAX) MissionName":

```suggestion
 m_sName "COTVT (2-45) Lamentin Losers"
```

Change 2 to whatever your minimum is